_dev/src/components/merchant-center-account/phone-verification/phone-verification-popin.vue

```diff
@@ -215,6 +244,42 @@ export default {
     getPhoneNumber() {
       this.phoneNumber = this.$store.getters['accounts/GET_SHOP_INFORMATIONS'].store.phone;
     },
+    resetVerificationCodeInputs() {
+      this.inputsVerificationCode = this.inputsVerificationCode
+        .map(() => ({
+          invitationId: null,
+        }));
+    },
+    goToNextInput(key) {
+      if (key.keyCode === 8) {
+        this.disableSendCodeButton = true;
+        this.isCodeValid = null;
+      }
+      const isFillCompletely = [];
+      this.inputsVerificationCode.forEach((code) => {
+        if (!code.invitationId) {
+          this.disableSendCodeButton = true;
+        } else {
+          isFillCompletely.push(code.invitationId);
+        }
+      });
+      if (key.keyCode !== 16 && (key.keyCode < 48 || key.keyCode > 57)) {
```

Comment:
To improve the readability f the code, have you tried looking at the `key` or `code` properties?

![image](https://user-images.githubusercontent.com/6768917/165913001-00622ffc-9855-4d2b-9dbb-614df6c1d01d.png)